packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -108,11 +112,50 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
   /* @conditional-compile-remove(rooms) */
   mobileWithPreview = mobileWithPreview && rolePermissions.cameraButton;
 
+  /* @conditional-compile-remove(call-readiness) */
+  const [isDrawerShowing, setIsDrawerShowing] = useState(true);
+  /* @conditional-compile-remove(call-readiness) */
+  const onLightDismissTriggered = (): void => {
+    // do nothing here
+    // only way to dismiss this drawer is clicking on allow access which will leads to device permission prompt
+  };
+  /* @conditional-compile-remove(call-readiness) */
+  const drawerStyle: _DrawerSurfaceStyles = {
+    root: {
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      width: '100%',
+      height: '100%',
+      // apply zindex = 99 so drawer appear over device buttons and other components in the config page
+      zIndex: 99
+    }
+  };
+
   return (
     <Stack className={mobileView ? configurationContainerStyleMobile : configurationContainerStyleDesktop}>
       <Stack styles={bannerNotificationStyles}>
         <ErrorBar {...errorBarProps} />
       </Stack>
+
+      {
+        /* @conditional-compile-remove(call-readiness) */
+        mobileView && isDrawerShowing && (
```

Comment:
if they've already accepted permissions and don't need to accept permissions, will this still always show?

Reply:
Yes same with enable mic/camera buttons